node/api/renter.go

```diff
@@ -138,12 +138,19 @@ type (
 
 	// DownloadInfo contains all client-facing information of a file.
 	DownloadInfo struct {
-		SiaPath     string    `json:"siapath"`
-		Destination string    `json:"destination"`
-		Filesize    uint64    `json:"filesize"`
-		Received    uint64    `json:"received"`
-		StartTime   time.Time `json:"starttime"`
-		Error       string    `json:"error"`
+		Destination     string `json:"destination"`     // The destination of the download.
+		DestinationType string `json:"destinationtype"` // Can be "file", "memory buffer", or "http stream".
+		Filesize        uint64 `json:"filesize"`        // DEPRECATED. Same as 'Length'.
```

Comment:
Should we rename `Filesize` to `V131Filesize` as an indicator that this field only exists for compatibility reasons and is deprecated?

Reply:
No, because if we change the name, we break compatibility. This field is exposed in the API, to preserve compatibility with existing apps we need to make sure they read the expected/correct value when they read the 'Filesize' field.

Reply:
I meant the field in the code, not the json tag.